emlparser/emlparser.py

```diff
@@ -69,13 +69,14 @@ def execute(self, request):
             kv_section = ResultSection('Email Headers', body_format=BODY_FORMAT.KEY_VALUE, parent=result)
 
             # Basic tags
-            if header.get('from', None):
+            if header.get('from', None) and re.match(EMAIL_REGEX, header['from'].strip()):
```

Comment:
rather than performing the strip method twice, assign it to a variable and use that